### PR TITLE
Solved #701: Added improved logged out message

### PR DESCRIPTION
**Pull request by PruthviThakor**
- changed logged out message to **You are not logged in, please login with `turso auth login` before running other commands.** as mentioned in issues #701 .

### CODE REVIEW

**Review by haaawk — COMMENTED** [review]
This will still prefix the message with "Error:" no?

**Review by haaawk — APPROVED** [review]

**Comment by haaawk** [comment]
> > This will still prefix the message with "Error:" no?
> 
> So It should be like normal message instead of error?

That's my understanding.

File: internal/cmd/db.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func getAccessToken(warnMultipleAccessTokenSources bool) (string, error) {
 		return envToken, nil
 	}
 	if !isJwtTokenValid(settingsToken) {
-		return "", fmt.Errorf("user not logged in, please login with %s", internal.Emph("turso auth login"))
+		return "", fmt.Printf("You are not logged in, please login with %s before running other commands.", internal.Emph("turso auth login"))
```
Comment (haaawk) [inline internal/cmd/db.go:115]: Does this even compile? Why are you returning string as an error?

**Review by haaawk — CHANGES_REQUESTED** [review]
I think returning string as an error is not the right way to fix the problem

**Comment by PruthviThakor** [comment]
> I think returning string as an error is not the right way to fix the problem

yeah I get that, We souldn't change Error to Print

**Comment by notrab** [comment]
I think it's fine to return a string instead of an error here. The original issue https://github.com/tursodatabase/turso-cli/issues/701 was to improve the wording by this kind of behaviour being common. Removing `error: ` prefix does that.

**Comment by haaawk** [comment]
#701 is properly fixed by https://github.com/tursodatabase/turso-cli/pull/851

**Pull request closed without merging**